packages/palette_generator/example/ios/Flutter/flutter_export_environment.sh

```diff
@@ -1,10 +1,11 @@
 #!/bin/sh
 # This is a generated file; do not edit or check into version control.
-export "FLUTTER_ROOT=/Users/gspencer/code/flutter"
-export "FLUTTER_APPLICATION_PATH=/Users/gspencer/code/packages/packages/palette_generator/example"
+export "FLUTTER_ROOT=/Users/mau/Development/Libraries/flutter"
```

Comment:
Can you please revert the changes to this file since they are unrelated to this PR? Thanks.

Reply:
Done!